src/renderer/components/ObservationCard.tsx

```diff
@@ -25,10 +27,12 @@ function ObservationCard(props: ObservationCardProps) {
     predictionOverride,
     onPredictionOverride,
     onPhotoClick,
+    lastObservationIndex,
```

Comment:
I found it a bit surprising that a single card needs to know how many cards are there in total. After reading the code I figured that it's to compute the previous/next card index.

This circles back to the idea of using separate components for maximized and non-maximized cards.
Consider this: a `MaximizedObservationCard` component with `onPrevious`, `onNext` and on `onClose` props. A couple other props could then be simplified / removed and the index computation could take place in the parent component, where the number of observations is readily available.